react/Viewer/Readme.md

```diff
@@ -86,3 +71,43 @@ const onFileChange = (file, nextIndex) => setState({ currentFileIndex: nextIndex
   </DemoProvider>
 </div>
 ```
+
+### Using a worker for pdfjs
+
+For performance reasons, it is important to use a web worker when showing PDF files in the viewer. If you use webpack, you should add the following alias in your configuration :
+
+```patch
++ resolve: {
++   alias: {
++     'react-pdf$' : 'react-pdf/dist/entry.webpack.js'
++   }
++ }
+```
+
+By default, the worker file will be created in a folder on a private route of your cozy application. If you need to display PDF files on a public page, you must explicitly load the web worker in your application like this:
```

Comment:
I don't understand this part ? Surely it does not mean react-router route ? In this case, I think it would be clearer if the doc would not mention route nor cozy application, it is just created in the build directory right ?

Reply:
> the worker file is just created in the build directory right ?

Yes that is correct. I'll use your suggestion ;)

Reply:
There, slightly tweaked. Let me know what you think :)